verilog/analysis/checkers/module_filename_rule.h

```diff
@@ -50,6 +51,9 @@ class ModuleFilenameRule : public verible::TextStructureLintRule {
   // Diagnostic message.
   static const char kMessage[];
 
+  // Ok to replace underscores with dashes in filenames.
```

Comment:
"Ok to treat underscores as dashes in filenames." ?

Reply:
Done (but other way around: treat dashes as underscores).